srtcore/queue.cpp

```diff
@@ -1257,11 +1235,12 @@ static string PacketInfo(const CPacket &pkt)
 
 EReadStatus CRcvQueue::worker_RetrieveUnit(ref_t<int32_t> r_id, ref_t<CUnit *> r_unit, sockaddr *addr)
 {
-#if !USE_BUSY_WAITING
-    // This might be not really necessary, and probably
-    // not good for extensive bidirectional communication.
-    m_pTimer->tick();
-#endif
+    // TODO: Put this change in a separate commit
+    //#if !USE_BUSY_WAITING
+    //    // This might be not really necessary, and probably
+    //    // not good for extensive bidirectional communication.
+    //    m_pTimer->tick();
+    //#endif
```

Comment:
- [ ] TODO: Put this change in a separate commit. It changes the synchronization logic.